src/globus_cli/commands/flows/list.py

```diff
@@ -98,7 +113,7 @@ def list_command(
     paginator = Paginator.wrap(flows_client.list_flows)
     flow_iterator = PagingWrapper(
         paginator(
-            filter_role=filter_role,
+            filter_roles=",".join(filter_role),
```

Comment:
Given that the SDK will comma-join, I'd generally prefer that we use `filter_roles=filter_role`.
However, I see that we have it annotated as `list[str]`, which is overly strict. I've put up a fix PR, but in the meantime, perhaps `filter_roles=list(filter_role)`?

Reply:
Yea, i should have changed this. i didn't revisit this code after adding commajoin in the other PR. 🤦